### PR TITLE
Add documentation for room and equipment endpoints

### PR DESCRIPTION
**Pull request by xoneco**
This is what it looks like:

image

### CODE REVIEW

**Comment by xoneco** [comment]
Review pls @uclapi/ucl-api-team

File: source/index.html.md [hunk 1]
```diff
@@ -126,6 +171,81 @@ bookingId | `String` | The of the booking
...
+Error | Description
+--------- | ---------
+`No token provided` | Gets returned when you have not supplied a `token` in your request.
+`Token does not exist` | Gets returned when you supply an invalid `token`.
+`No roomid supplied` | Gets returned when you don't supply a `roomid`.
+`No siteid supplied` | Gets returned when you don'y supply a `siteid`.
+
+# Contributing
+These docs are open sourced at [https://github.com/uclapi/apiDocs](https://github.com/uclapi/apiDocs).  
+Any and all contributions are welcome! If you spot a type or error, fix it :)
```
Comment (hennersz) [inline source/index.html.md]: should it say typo here?

Comment (xoneco) [inline source/index.html.md]: Yes :P @abdulfaizp could you update this in your PR?

Comment (hennersz) [inline source/index.html.md]: how ironic

Comment (xoneco) [inline source/index.html.md]: oh yeah. lol 
i didn't even realise

Maybe we should leave it in?

**Pull request merged**